src/analysis/normed_space/basic.lean

```diff
@@ -610,13 +610,18 @@ begin
   exact (tendsto_inv hx)
 end
 
+namespace real
+
 instance : normed_field ℝ :=
 { norm := λ x, abs x,
   dist_eq := assume x y, rfl,
   norm_mul' := abs_mul }
 
 instance : nondiscrete_normed_field ℝ :=
 { non_trivial := ⟨2, by { unfold norm, rw abs_of_nonneg; norm_num }⟩ }
+
+end real
+
 end normed_field
```

Comment:
```suggestion
end normed_field

instance : normed_field ℝ :=
{ norm := λ x, abs x,
  dist_eq := assume x y, rfl,
  norm_mul' := abs_mul }

instance : nondiscrete_normed_field ℝ :=
{ non_trivial := ⟨2, by { unfold norm, rw abs_of_nonneg; norm_num }⟩ }
```

While we're on the topic of namespaces. The autogenerated names for instances are a little confusing. But I think this should name these instances `real.normed_field` and `real.nondiscrete_normed_field`, instead of `normed_field.real.normed_field`.

Reply:
Thanks.  Yes, that's what I was trying to achieve.